src/props.animations.js

```diff
@@ -77,20 +77,20 @@ export default {
   60% { transform: translateY(-5%) }
   80% { transform: translateY(5%) }
 }`,
-  "--animation-spin": "spin 2s linear infinite", 
+  "--animation-spin": "spin var(--spin-duration, 1ms) linear infinite", 
   "--animation-spin-@": `
 @keyframes spin {
   to { transform: rotate(1turn) }
 }`,
-  "--animation-ping": "ping 5s var(--ease-out-3) infinite", 
+  "--animation-ping": "ping var(--ping-duration, 1ms) var(--ease-out-3) infinite", 
   "--animation-ping-@": `
 @keyframes ping {
   90%, 100% {
     transform: scale(2);
     opacity: 0;
   }
 }`,
-  "--animation-blink": "blink 1s var(--ease-out-3) infinite", 
+  "--animation-blink": "blink var(--blink-duration, 1ms) var(--ease-out-3) infinite", 
```

Comment:
```suggestion
  "--animation-blink": "blink var(--blink-duration, 2s) var(--ease-out-3) infinite", 
```

for this one, let's go a bit more custom and double the duration, so it's a non-threatening but still effective blink. i felt stepped animations were much were than just allowing the blink crossfade.